test/EVMHost.h

```diff
@@ -180,7 +180,8 @@ class EVMHost: public evmc::Host
 	static evmc::result resultWithGas(evmc_message const& _message, bytes const& _data) noexcept;
 
 	evmc::VM* m_vm = nullptr;
-	evmc_revision m_evmVersion;
+	langutil::EVMVersion m_evmVersion;
+	evmc_revision m_evmRevision;
```

Comment:
Could you add two comments explaining the difference between revision and version?